src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -144,11 +151,64 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
 	@IBOutlet private weak var exposureHistoryNoticeImageView: UIImageView!
 	@IBOutlet private weak var exposureHistoryTitleLabel: ENALabel!
 	@IBOutlet private weak var exposureHistoryDetailLabel: ENALabel!
-	
+
+	// PCR & Antigen TestsStackView
+	@IBOutlet private weak var testsStackView: UIStackView!
+
 	// Check-Ins with risk
 	@IBOutlet private weak var checkinHistoryStackView: UIStackView!
 	@IBOutlet private weak var checkinHistoryNoticeImageView: UIImageView!
 	@IBOutlet private weak var checkinHistoryTitleLabel: ENALabel!
 	@IBOutlet private weak var checkinHistoryDetailLabel: ENALabel!
 	@IBOutlet private weak var checkinsWithRiskStackView: UIStackView!
+
+	private func arrangedView(for test: DiaryDayTest) -> UIView {
+		let containerView = UIView()
+		containerView.translatesAutoresizingMaskIntoConstraints = false
+
+		let separator = UIView()
+		separator.translatesAutoresizingMaskIntoConstraints = false
+		separator.backgroundColor = .enaColor(for: .hairline)
+		containerView.addSubview(separator)
+
+		let imageView = UIImageView()
+		imageView.translatesAutoresizingMaskIntoConstraints = false
+		imageView.contentMode = .center
+		imageView.image = test.result == .negative ? UIImage(imageLiteralResourceName: "Test_green") : UIImage(imageLiteralResourceName: "Test_red")
+
+		let titleLabel = ENALabel()
+		titleLabel.style = .body
+		titleLabel.text = test.type == .pcr ? AppStrings.ContactDiary.Overview.Tests.PCRRegistered : AppStrings.ContactDiary.Overview.Tests.AntigenDone
+
+		let detailLabel = ENALabel()
+		detailLabel.style = .subheadline
+		detailLabel.textColor = .enaColor(for: .textPrimary2)
+		detailLabel.text = test.result == .negative ? AppStrings.ContactDiary.Overview.Tests.negativeResult : AppStrings.ContactDiary.Overview.Tests.positiveResult
+
+		let verticalStackView = UIStackView(arrangedSubviews: [titleLabel, detailLabel])
```

Comment:
missing translatesAutoresizingMaskIntoConstraints = false ?

Reply:
when adding it to a stackView this is not needed